app/controllers/admin/book_controller.rb

```diff
@@ -0,0 +1,17 @@
+class Admin::BookController < Admin::BaseController
+  before_action :find_book, except: :index
+
+  def show
+    @authors = @book.book_authorships
```

Comment:
```suggestion
    @authors = @book.authors
```
ri là dc nè
tìm hiểu thêm config quan hệ nhiều nhiều trong rails nhé

Reply:
okie chị nè